src/view/configuration/configuration.jsx

```diff
@@ -345,7 +354,7 @@ const Configuration = () => {
                                 name={`instances.${index}.name`}
                                 component={Textfield}
                                 componentClassName="u-fieldLong"
-                                supportDataElement
+                                supportDataElement="replace"
```

Comment:
Do you mind taking a look at the other `supportDataElement` usage within the extension and seeing if they should use `supportDataElement="replace"` as well? For example, it would make sense for all the usage in the Send Event action to be `replace` as well. To manually test the actions, read this first: https://github.com/adobe/reactor-extension-alloy#manually-testing-views-using-the-local-sandbox

Feel free to do this in a separate PR if you'd like